mmdet3d/models/voxel_encoders/pillar_encoder.py

```diff
@@ -82,14 +82,16 @@ def __init__(self,
         self.y_offset = self.vy / 2 + point_cloud_range[1]
         self.point_cloud_range = point_cloud_range
 
-    def forward(self, features, num_points, coors):
+    def forward(self, features, num_points, coors, legacy=True):
```

Comment:
legacy should not be added here, it should be an attribute of the module.

Reply:
just out of curiosity, what is the difference between legacy or not? The output is the same or I miss sth?